shared/types/form/form.ts

```diff
@@ -163,6 +165,7 @@ export interface StorageFormBase extends FormBase {
   payments_channel: FormPaymentsChannel
   payments_field: FormPaymentsField
   business?: FormBusinessField
+  newEncryptionBoundary?: boolean
```

Comment:
Since FormBase already has `newEncryptionBoundary?: boolean`, no longer necessary to define here

Reply:
Good point, fixed typings in [b23f39b](https://github.com/opengovsg/FormSG/pull/6595/commits/b23f39be4bf5b3c70fb21b52cee10f4397438f37)!